rs/nns/governance/unreleased_changelog.md

```diff
@@ -4,14 +4,15 @@ In general, upcoming/unreleased behavior changes are described here. For details
 on the process that this file is part of, see
 `rs/nervous_system/changelog_process.md`.
 
-
```

Comment:
All other unreleased_changelog.md files have this. I like it this way; regardless, what really matters is consistency.